### ISSUE TITLE
Fan Search/Filtering/Sort - Org members should be able to search across all customers

### ISSUE THREAD

**Issue by zmelamed** [big-neon/bigneon#53]
Under "Fans" in studio, an org member should be able to search all fans by name and email

Fans table should be sortable by all columns

**Comment by Timo614** [comment]
@zmelamed this is for any organization roles outside of the door person and box office right? Or should box office users be able to search by all customers?

Also can you define which customers should be included in the results?
- Users who purchase orders for an event_id belonging to the organization
- Users who have no orders but have an ticket belonging to a ticket type owned by the organization (transferred in)

Should it include both? Or should we not include any subset of the customers? For example should we include users who later transferred out their ticket? They do have an associated order still just no longer have any access.

For now I'm moving forward assuming both but can adjust as needed via feedback.

**Comment by Timo614** [comment]
While implementing this I noticed this already exists via: https://github.com/big-neon/bn-api/blob/master/db/src/models/organizations.rs#L592

`/organizations/{id}/fans`

Should be set on the bn-api side so likely only needs the bn-web and bn-api-node.

**Comment by Krakaw** [comment]
@Timo614 you're right there is that endpoint. This will be addressed in big-neon/bn-web#1257

**Comment by zmelamed** [comment]
Is there still a pending question on this one? @Timo614

**Comment by Timo614** [comment]
Nope. I'll remove the label.  :+1:
